src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/reserved-type-names.txt

```diff
@@ -1,315 +1,2980 @@
+AbstractiveSummarizeAction
```

Comment:
Should we add a test that validates the files? (ensures that they are sorted and match line for line by type name)

Reply:
the sorting is checked: https://github.com/Azure/azure-sdk-tools/blob/main/src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/DuplicateTypeNameAnalyzer.cs#L88